pkg/formats/influx/influx.go

```diff
@@ -357,3 +364,39 @@ func influxEscape(s string) string {
 		return s
 	}
 }
+
+func getMib(attr map[string]interface{}, ip interface{}) string {
+	mib, ok := attr["mib-name"].(string)
+	if !ok {
+		return "none"
+	}
+	delete(attr, "mib-name")
+	if mib == "" {
+		mib = "none"
+	}
+
+	// Also remove any lingering droppable fields.
+	for k, v := range attr {
+		if _, ok := dropFields[k]; ok {
+			delete(attr, k)
+		}
+		if k == "Index" {
+			delete(attr, k)
+			attr["index"] = v
+		}
+	}
+	if ip != nil {
+		attr["device_ip"] = ip
+	}
+
+	return mib
+}
+
+var dropFields = map[string]bool{
+	//"device_name": true,
+	"eventType":   true,
+	"SysObjectID": true,
+	"provider":    true,
+	"SysServices": true,
+	"SysDescr":    true,
```

Comment:
We don't need SysDescr and SysObjectID to be reported in every line, but we do need them reported sometime or occasionally. Not sure what the best way to manage that would be.

Reply:
In NR these are attached to uptime metrics and only these.